engine/events/unown_walls.asm

```diff
@@ -105,8 +105,8 @@ DisplayUnownWords:
 	and a
 	jr z, .load
 
-	ld d, $0
-	ld e, $5
+	ld d, 0
+	ld e, 5
```

Comment:
Would it be worth defining a constant? I know it sems obvious what the value is used for.

```
DEF UNOWNWALL_MENU_HEADER_LENGTH EQU 5
```

```suggestion
	ld d, HIGH(UNOWNWALL_MENU_HEADER_LENGTH)
	ld e, LOW(UNOWNWALL_MENU_HEADER_LENGTH)
```

```
; entries correspond to UNOWNWORDS_* constants
	table_width UNOWNWALL_MENU_HEADER_LENGTH, MenuHeaders_UnownWalls
	for x, 1, NUM_UNOWN_WALLS + 1
		db MENU_BACKUP_TILES ; flags
		DEF n = CHARLEN("{UNOWNWORD_{d:x}}")
		menu_coords 9 - n, 4, 10 + n, 9
	endr
	assert_table_length NUM_UNOWN_WALLS
```